app/helpers/sufia_helper.rb

```diff
@@ -15,4 +15,18 @@ def link_to_fields(fields, field_string)
     fields.map { |tag| link_to_facet(tag, field_string) }
   end
 
+  def link_to_field(fieldname, fieldvalue, displayvalue = nil)
```

Comment:
Yeah I think this was for an older way I was unpacking the rdf objects, trying to reuse the changes trey had made. But in the partials, im not using link to field, but link to facet. So I can change this back and it should be fine.